### DISCUSSION TITLE
How to select existing items when adding blocks?

### DISCUSSION THREAD

**Discussion by jdjoshuadavison** [formfcw/directus-extension-flexible-editor#19] [Q&A] [answered]
**Describe the bug**
When adding a block using the Flexible Editor, it goes straight to the `Creating X in Xs` drawer and does not give me the option to select an existing item.

https://github.com/formfcw/directus-extension-flexible-editor/assets/34007858/6fc0f28d-0562-4ee7-811a-4e46970a6aaa

**To Reproduce**
Install flexible editor extension in a new project and follow the `usage-with-relation-nodes.mp4` video to set it up.

**Flexible Editor Version:**
1.0.3

**Directus Version:**
10.4.3

**Comment by formfcw** [comment, marked as answer]
Thanks for the question! I converted this to a Q&A, as this is the correct behavior.

The correct way to do this is to create a collection like `related_items` and use this instead. In this new collection you then create a M2M or O2M (or M2A) field that relates to your `items` collection.

I then would suggest putting those kind of collections used by Flexible Editor into a folder called `Editor` or `Editor Collections` or whatever. This keeps the structure of your data model readable. Check out the video (at the beginning) in this section of the README: https://github.com/formfcw/directus-extension-flexible-editor#usage-with-relation-nodes

**Reply by jdjoshuadavison** [reply]
Hmm, that is the video I followed step by step, twice. Unfortunately I'm not going to be able to look at this until Thursday - but hopefully I'll be able to work it out. I'll get back to you Thursday/Friday if I can't!

**Reply by jdjoshuadavison** [reply]
Oookay. I got it. Thanks so much for this.

**Reply by jablonski** [reply]
That solves the problem, but the setup is quite hard. Maybe you can create a short video how to setup this use case? Would help a lot.

**Reply by smarchesini-DQ** [reply]
Could you explain how to do this with an example please?  I want to be able to use existing collection items but can not figure out exactly how to do that. TIA

**Reply by formfcw** [reply]
I’ve created a video tutorial: https://github.com/formfcw/directus-extension-flexible-editor#tutorial-relation-node-with-existing-items

**Reply by jablonski** [reply]
Very nice and much appreciated! Thank you very much.
